include/tvm/ir/memory_pools.h

```diff
@@ -60,10 +59,12 @@ struct PoolInfoNode : public Object {
    */
   bool is_internal = false;
 
+  /*! \brief The targets linked to the pool */
+  Array<Target> targets;
```

Comment:
just curious why rw/ro went away?

Reply:
@areusch RW/RO concept seemed redundant after introduction of the concept of pools (we have discussed this here in the PR -- I'll post an update to the RFC after these are settled).

Initially, it was proposed in the absense of the in-direction provided by the memory pools (vs memories). Now, since we have that in-direction we dont need to mix RW/RO in a single pool. Therefore it felt simpler just to use two seperate pools for each and still place them in the same memory to get the same effect.